test/integration/sonobuoy_integration_test.go

```diff
@@ -9,13 +9,16 @@ import (
 	"io/ioutil"
 	"os"
 	"os/exec"
+	"path/filepath"
 	"strings"
 	"testing"
 	"time"
 )
 
 const (
 	defaultSonobuoyPath = "../../sonobuoy"
+	bash                = "/bin/sh"
```

Comment:
My preference here would be to use `/bin/bash` here to avoid unexpected differences between the shells. I think we've been caught out before by differences between these two. I think it's typically symlinked to different shells on different systems, dash on Linux (and most of the container images we use) and bash on macOS.